Awful.apk/src/main/java/com/ferg/awfulapp/ForumsPagerController.kt

```diff
@@ -135,16 +134,6 @@ class ForumsPagerController(
     }
 
 
-    /**
```

Comment:
Can you revert the deletions in this file? Even if an activity weren't using these callbacks right now, this is all state management wiring that only exposes actual page changes, deferring them until the page fragment is actually created if necessary. The idea behind this controller class is to keep all the internal viewpager wrangling hidden and managed safely